app/root/root.tsx

```diff
@@ -23,6 +23,8 @@ interface State {
   preferences: UserPreferences;
 }
 
+capabilities.register("BuildBuddy Community Edition", false, [Path.invocationPath]);
```

Comment:
fyi, moved the capabilities registration here so that `UserPreferences` can read them in the constructor